app/cortexlib/ipblockcheck.py

```diff
@@ -36,13 +32,10 @@ def extract_ip_from_event(cls, event):
                 pass
         return None
 
-
     def set_ipblockcheck_config(self):
-        with open(IpBlockCheck.config_file, 'r') as config:
-            ipblock_conf = yaml.load(config)["ipblocker"]
-        self.ip_zone_name = ipblock_conf["ip_zone_name"]
-        self.trigger_events = ipblock_conf["trigger_events"]
-        self.trigger_only_on_critical = ipblock_conf["trigger_only_on_critical"]
+        self.ip_zone_name = os.getenv("IPBLOCKER_IP_ZONE_NAME")
```

Comment:
set_config is similar in the other test. Is there a way to do it via a lib that'll set the variables in the init.